aws_securityhub/icon_aws_securityhub/connection/connection.py

```diff
@@ -23,5 +23,5 @@ def connect(self, params):
         self.aws = boto3.Session(**kwargs)
 
     def test(self):
-        client = self.aws.client('securityhub')
-        return {}
+        self.aws.client('securityhub')
+        pass
```

Comment:
You don't need `pass` unless the function does not do anything and is merely a stub.